### ISSUE TITLE
[BUG] rspamadm configtest complains about symbol DCC_CHECK having no score registered

### ISSUE THREAD

**Issue by JoeKun** [rspamd/rspamd#3594]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Read about bug reporting in general: https://rspamd.com/doc/faq.html#how-to-report-bugs-found-in-rspamd
    * Enabled relevant debugging logs: https://rspamd.com/doc/faq.html#how-to-debug-some-module-in-rspamd 
    * Checked the FAQs about Core files in case of fatal crash: https://rspamd.com/doc/faq.html#how-to-figure-out-why-rspamd-process-crashed
    * Tried ASAN package and obtained the ASAN report (if possible): https://rspamd.com/doc/faq.html#asan-builds 
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+user%3Arspamd
    * Checked that there is not already an experimental package or master branch

**Describe the bug**
`rspamadm configtest` complains about symbol `DCC_CHECK` having no score registered

### Steps to Reproduce

1. Configure `rspamd` and enable the `dcc` module
2. Run `rspamadm configtest -c /usr/local/etc/rspamd/rspamd.conf`
3. Output looks like:

```
2021-01-03 21:30:12 #0(main) <3zasmg>; symcache; rspamd_symcache_validate_cb: symbol DCC_CHECK has no score registered, skip its check
syntax OK
```

**Expected behavior**
I expected concise output about the syntax of the configuration, without any warnings about any symbols missing a score registered.

In other words, I expected the output of that `configtest` command to look like:

```
syntax OK
```

### Versions

```
# rspamd --version
Rspamd daemon version 2.6
```

```
# uname -a
FreeBSD REDACTED_HOSTNAME 12.2-RELEASE-p1 FreeBSD 12.2-RELEASE-p1 GENERIC  amd64
```

Here's additional information on how the `rspamd` port was compiled:

```
# pkg show rspamd
rspamd-2.6
Name           : rspamd
Version        : 2.6
Installed on   : Mon Nov  2 15:28:07 2020 PST
Origin         : mail/rspamd
Architecture   : FreeBSD:12:amd64
Prefix         : /usr/local
Categories     : mail
Licenses       : APACHE20
Maintainer     : vsevolod@FreeBSD.org
WWW            : https://rspamd.com/
Comment        : Fast spam filtering system
Options        :
	BLAS           : off
	CGP            : off
	GPERF          : off
	HYPERSCAN      : on
	LUAJIT         : on
Shared Libs required:
[...]
```

### Additional Information

Here's my configuration for rspamd's `dcc` module, at `/usr/local/etc/rspamd/local.d/dcc.conf`:

```
enabled = true;
socket = "/var/db/dcc/dccifd";
timeout = 5.0;
```

**Comment by vstakhov** [comment]
Please check the faq entry about scores and symbols. Not an Rspamd issue.

**Comment by JoeKun** [comment]
I read the [FAQ entry about changing the score for a symbol](https://rspamd.com/doc/faq.html#how-to-change-score-for-some-symbol), and I gave that a try.

After putting the following:

```
symbols {
    "DCC_CHECK" {
        weight = 0.0;
    }
}
```

in `/usr/local/etc/rspamd/local.d/groups.conf`, the warning was gone.

That said, I respectfully disagree with your characterization that this is not an Rspamd issue. Just because I can easily workaround this issue using the very extensible configuration system doesn't mean that every administrator who installs Rspamd and enables the `dcc` module should face this warning. I think there's something to be said about making the "out of box experience" as smooth as possible, and remove head scratchers like this one.

I believe this can actually be resolved in a way that doesn't require any configuration hack like the one I mentioned above. Here's a patch I just experimented with, which seems to work for me.

```
diff --git a/src/plugins/lua/dcc.lua b/src/plugins/lua/dcc.lua
index 4fe7bfe78a31..d5cb47372620 100644
--- a/src/plugins/lua/dcc.lua
+++ b/src/plugins/lua/dcc.lua
@@ -88,6 +88,10 @@ if rule then
     parent = id,
     name = 'DCC_FAIL'
   }
+  rspamd_config:set_metric_symbol({
+    score = 0.0,
+    name = 'DCC_CHECK',
+  })
   rspamd_config:set_metric_symbol({
     group = N,
     score = 1.0,
```

@vstakhov How would you feel about such a change to the `dcc.lua` plugin? I'd be more than happy to create a pull request with this change.

**Comment by vstakhov** [comment]
Yes, you are right. DCC_CHECK should be a callback symbol as far as I see. I will fix it. Sorry for confusion.

**Comment by JoeKun** [comment]
Thanks @vstakhov! I can confirm this is fixed in rspamd 2.7.
